domains/meywy.json

```diff
@@ -0,0 +1,14 @@
+{
+    "description": "Portfolio",
+    "repo": "https://github.com/Meywy/meywy.github.io",
+    "owner": {
+        "username": "Meywy",
+        "email": "meywyy@gmail.com",
+        "discord": "923162990419791913"
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```

Comment:
Error 404 Not Found

Reply:
Hey, I dont quite understand, the website https://meywy.netlify.app is accessible normally.

Reply:
He most probably visited the IP instead of the link